sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -1522,6 +1549,41 @@ private Mono<Void> sendMessage(ServiceBusMessage message) {
         });
     }
 
+    private Mono<Void> sendMessages(List<ServiceBusMessage> messages) {
+        final Iterator<ServiceBusMessage> messagesItr = messages.iterator();
+        if (messagesItr.hasNext()) {
+            return sendNextBatch(messagesItr.next(), messagesItr);
+        } else {
+            return Mono.empty();
+        }
+    }
+
+    private Mono<Void> sendNextBatch(ServiceBusMessage first, Iterator<ServiceBusMessage> messagesItr) {
```

Comment:
I wonder if adding this is necessary? I feel like it adds more points of failure if we have to analyse both the test case logic and the product code.

Reply:
We just improved the send API in the SenderClient in this [pr](https://github.com/Azure/azure-sdk-for-java/pull/40462), we can use that API here. I'll rebase, use it and remove this send test method.

Reply:
done